srcpkgs/NetAuth-ldap/files/netauth-ldap/run

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+[ -r ./conf ] && . ./conf
```

Comment:
Does this affect anything if it's not used in the exec line?

Reply:
Yes, as you can set the log level via an environment variable.